scripts/staging/slicing/datasets/insurance.csv

```diff
@@ -0,0 +1,1339 @@
+age,sex,bmi,children,smoker,region,charges
```

Comment:
We usually try not to include these datasets, as the potentially have licenses that forbid redistribution. Maybe you could automatically fetch these datasets in your tests?